.lgtm.yml

```diff
@@ -0,0 +1,2 @@
+queries:
+  - exclude: cs/campaign/constantine
```

Comment:
@grendello: what's *this* do?

Reply:
It disables the particular query/check globally, for the entire repo. This one was a campaign asking for contact with regards to certain constructs, not an error or warning.